### ISSUE TITLE
Add a "Show Getting Started" checkbox to the user manager and My Account screens

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19798]
We need to add a checkbox to the user manager and My Account Screens called "Show Getting Started". If it is checked, then we call the api endpoint to add the portlet to the user.  If unchecked, then we remove it.

Add the checkbox here:

![Screen Shot 2021-01-12 at 3.31.16 PM.png](https://images.zenhubusercontent.com/56e187366caf56c9d38d8bc5/6c081a79-f9ef-48be-937b-3b02fabd88c7)

And here:
![Screen Shot 2021-01-12 at 3.33.15 PM.png](https://images.zenhubusercontent.com/56e187366caf56c9d38d8bc5/68bf76e9-7b10-4bad-8169-6b2aa54292ec)

**Comment by alfredo-dotcms** [comment]
FETCH to set starter page:
```
fetch('/api/v1/toolgroups/gettingstarted/_addtocurrentuser', {
  method: 'PUT',
  headers: {
    'Content-type': 'application/json; charset=UTF-8'
  }
})
.then(res => res.json())
.then(console.log)

fetch('/api/v1/toolgroups/gettingstarted/_addtocurrentuser', {
  method: 'PUT',
  headers: {
    'Content-type': 'application/json; charset=UTF-8'
  }
})
.then(res => res.json())
.then(console.log)
```

**Comment by alfredo-dotcms** [comment]
![image](https://user-images.githubusercontent.com/37185433/104650050-409e1480-567b-11eb-81b1-fb4c05aea02c.png)

this won't be implemented on this page, right now the endpoint does not support another user rather than the current logged in, so this will be addressed on another task

**Comment by alfredo-dotcms** [comment]
core-web PR: https://github.com/dotCMS/core-web/pull/1552

**Comment by nollymar** [comment]
**Internal QA:** Passed.

The functionality works as expected. 

<img width="1518" alt="Screen Shot 2021-01-25 at 12 44 18 PM" src="https://user-images.githubusercontent.com/8741395/105744459-25e55e80-5f0b-11eb-8ec2-0e2389d6045a.png">


I left an observation on a related task: https://github.com/dotCMS/core/issues/19802#issuecomment-766991514

**Comment by nollymar** [comment]
**Pending task for the Next Release:** turn on the Getting Started checkbox for each user in the full and empty starters and generate new starters with this change.

**Comment by wezell** [comment]
The "Show Getting Started" checkbox should be under the Email Address field and not under the password fields.

**Comment by nollymar** [comment]
PR with changes suggested by @wezell : https://github.com/dotCMS/core-web/pull/1571

**Comment by bryanboza** [comment]
If you enable this checkbox, we are not detecting this as a change, this because the "Save" button still looking disabled, we need to enable this button to allow save the changes, also if you enable this and then cancel, we need to discard the changes. 
![image](https://user-images.githubusercontent.com/2641437/108923577-b1333c00-75fe-11eb-8256-192af6b40f2f.png)

**Comment by alfredo-dotcms** [comment]
PR: https://github.com/dotCMS/core-web/pull/1595/files

**Comment by alfredo-dotcms** [comment]
Note to QA: to make enable the form to SAVE the user needs to put all the required fields, so the `Current Password` needs to be filled to be able to save the changes

**Comment by nollymar** [comment]
**Passed internal QA:**

<img width="1449" alt="Screen Shot 2021-03-03 at 11 23 57 AM" src="https://user-images.githubusercontent.com/8741395/109837246-fa911600-7c12-11eb-8bf0-fb6519d20ea5.png">

**Comment by bryanboza** [comment]
Fixed, tested on release-21.03 // Postgres // FF
